markhor_flippers/src/markhor_flippers.cpp

```diff
@@ -203,6 +255,11 @@ int main(int argc, char** argv)
     cm.update(time, period);
     hw.write();
 
+    publishFRTarget(hw.getFRTarget());
```

Comment:
I incline to say that it should be inside the hardware interface object instead of outside. And when we are `read`ing the hw object, we could read->publish the values. What do you think ?

Reply:
Donc je met tous les publish dans `markhor_hw_interface_flippers.cpp` ?

Reply:
Well, I'm asking, what do you think. I don't want to dictate to you what to do, and I want you to think. 

Now, I'm saying that I think that the publishing could be inside the class since that's where the data is. Right now, you're asking for the data and then publish. That seems like unnecessary steps.

Reply:
Also, as a precision, when I'm saying : 
> read->publish
I'm saying to when we do the read, we could inside the read function publish.

Reply:
C'est vrai que c'est des étapes de plus, j'avais mis les publish dans `markhor_flippers.cpp` pour regrouper avec tous les autres publish (mode et controller) et utiliser `markhor_hw_interface_flippers.cpp` pour les `ROS_INFO_THROTTLE`
Après, je sais pas ce qui est le mieux entre les deux options

Reply:
Well, I'll put the publish inside the function read in the ` MarkhorHWInterfaceFlippers` class.